docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -113,5 +113,13 @@ The following examples show how to use all the previous methods:
 |    set FASTDDS_STATISTICS=HISTORY_LATENCY_TOPIC;ACKNACK_COUNT_TOPIC;DISCOVERY_TOPIC;PHYSICAL_DATA_TOPIC              |
 +----------------------------------------------------------------------------------------------------------------------+
 
+.. note::
+
+    These are all the statistics topics:
+
+    .. code-block:: bash
+
+        HISTORY_LATENCY_TOPIC;NETWORK_LATENCY_TOPIC;PUBLICATION_THROUGHPUT_TOPIC;SUBSCRIPTION_THROUGHPUT_TOPIC;RTPS_SENT_TOPIC;RTPS_LOST_TOPIC;HEARTBEAT_COUNT_TOPIC;ACKNACK_COUNT_TOPIC;NACKFRAG_COUNT_TOPIC;GAP_COUNT_TOPIC;DATA_COUNT_TOPIC;RESENT_DATAS_TOPIC;SAMPLE_DATAS_TOPIC;PDP_PACKETS_TOPIC;EDP_PACKETS_TOPIC;DISCOVERY_TOPIC;PHYSICAL_DATA_TOPIC
```

Comment:
I suggest instead of listing all the topic names, to put a link to the API reference: https://fast-dds.docs.eprosima.com/en/latest/fastdds/api_reference/statistics/topic_names.html